### ISSUE TITLE
What moments to choose for mining?

### ISSUE THREAD

**Issue by VK-VZ** [UnamSanctam/SilentCryptoMiner#1957]
Hi all.  Please tell me which coins are more profitable to mine now.  I have xmr and etc, but the profitability is very low.  Maybe it’s more profitable to mine something else?

**Comment by UnamSanctam** [comment]
Right now Zephyr is the most profitable CPU coin, for GPU coins you can check https://whattomine.com/.

**Comment by VK-VZ** [comment]
Thanks for the quick response.  Please advise you are a gpu miner.  And you have examples of settings in the wiki for xmr and etc.  Can you make 2 more examples of settings for Zephyr and gpu miner?  I trust you, I'm just afraid to insert the wrong settings

> На данный момент Zephyr — самая прибыльная монета для процессоров (тот же алгоритм, что и Monero), а для монет для графических процессоров вы можете проверить https://whattomine.com/ .

**Comment by VK-VZ** [comment]
Is it profitable to mine Clore(CLORE) now?  Did I understand correctly?
![Screenshot_20231122_024004_Chrome](https://github.com/UnamSanctam/SilentCryptoMiner/assets/149912866/cee4fb04-65ad-4ecc-80f2-7e71bf33f5ff)

**Comment by UnamSanctam** [comment]
Yes though the profitability of the GPU coins change all the time so any coin in the top 10 usually works. To change it to mine Zephyr and some other GPU coin then you only need to change the pool and the wallet (and the algorithm for the GPU miner). Try to find some pool you like for Zephyr and whatever GPU coin you choose to mine and then post them here and I can help. You can also mine using coin-switching pools like Zergpool to do that automatically but Zergpool can be quite complicated.
